schema.sql

```diff
@@ -65,3 +65,6 @@ vet_clinic(# date_of_visit DATE,
 vet_clinic(# FOREIGN KEY (vet_id) REFERENCES vets(id),
 vet_clinic(# FOREIGN KEY (animal_id) REFERENCES animals(id)
 vet_clinic(# );
+           
```

Comment:
- [ OPTIONAL] Your execution times greatly improved, but you could try adding indexes on foreign keys or columns frequently used as below and try out for different execution times.
  `CREATE INDEX ON visits (animal_id);`
  `CREATE INDEX ON visits (vet_id);`
  `CREATE INDEX ON owners (email);`

Reply:
Hi @t-yanick !! 
Thanks for the approval. The creation of indexes exactly on the columns you specified helped me reduce the execution time in the first place. Sorry for not including it in the PR. Thank you!